lib/ansible/cli/galaxy.py

```diff
@@ -262,22 +262,22 @@ def add_init_options(self, parser, parents=None):
         init_parser.add_argument('--init-path', dest='init_path', default='./',
                                  help='The path in which the skeleton {0} will be created. The default is the '
                                       'current working directory.'.format(galaxy_type))
-        init_parser.add_argument('--{0}-skeleton'.format(galaxy_type), dest='{0}_skeleton'.format(galaxy_type),
+        init_parser.add_argument('--skeleton', '--{0}-skeleton'.format(galaxy_type), dest='{0}_skeleton'.format(galaxy_type),
```

Comment:
Please create a [changelog fragment](https://docs.ansible.com/ansible/latest/community/development_process.html#creating-a-changelog-fragment). See [this fragment](https://github.com/ansible/ansible/blob/c2f528b7686c9573aa8493357219e0bbe4416dd3/changelogs/fragments/reboot-change-default-boot-command.yaml) as an example.

Reply:
@Akasurde  Added changelog fragments! Thanks for reviewing.